pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -52,6 +52,20 @@ func validateBindingSpec(spec *sc.BindingSpec, fldPath *field.Path, create bool)
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("secretName"), spec.SecretName, msg))
 	}
 
+	if spec.AlphaPodPresetName != nil {
+		for _, msg := range apivalidation.NameIsDNSSubdomain(*spec.AlphaPodPresetName, false /* prefix */) {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("alphaPodPresetName"), spec.AlphaPodPresetName, msg))
+		}
+
+		if spec.AlphaPodPresetSpec == nil {
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("alphaPodPresentName"), "alphaPodPresetName may not be specified with alphaPodPresetSpec"))
```

Comment:
Maybe my brain is fried, but shouldn't this be "alphaPodPresetSpec must be specified with alphaPodPresetName"?